contrib/helm/harbor/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 name: harbor
-version: 0.0.1
-appVersion: 1.3.0
-description: An Enterprise-class Docker Registry Harbor by VMware
+version: 0.2.0
```

Comment:
we shouldn't jump a minor version here.  it should either go to `0.1.0` or `1.0.0`.

Reply:
Will use 0.1.0.